src/ProxyClient/Varnish.php

```diff
@@ -151,6 +160,8 @@ protected function configureOptions()
         $resolver->setDefaults([
             'tags_header' => self::DEFAULT_HTTP_HEADER_CACHE_TAGS,
             'header_length' => 7500,
+            'tags_invalidation_by_purge' => false,
+            'tags_separator' => ',',
```

Comment:
ah, okay. we used comma separated until now, yes. as ban is based on regex, it could be anything as long as escaping happens correctly. its all on php side, varnish just stores the header and runs a regex. (actually, if escaping does not happen, we only risk to ban too much, which is only a performance loss but should not be too likely to really happen and thus no big deal)

imho we can't change to xkey as default until we drop support for all varnish versions that come without xkey. but the doc can clearly state that if you are lucky enough to have a varnish recent enough that it supports xkey, you should switch to that.

i wonder however if we really want to make the separator configurable. i would prefer to check if xkey is enabled and if so pick the right separator. or simply switch to a whitespace as separator instead of the comma. actually i could see switching to whitespace even with ban, if you can do that for 2.0.0. afaik its not even specified in the doc how we separate so its really up to us.